e2e-tests/portal/cypress/support/portal-api/deleteDeals.js

```diff
@@ -4,11 +4,11 @@ const deleteAllDeals = (token, deals) => {
   if (!deals || !deals.length) return;
 
   deals.forEach(async (deal) => {
-    if (deal.dealType === 'BSS/EWCS') {
+    if (deal.product === 'BSS/EWCS') {
```

Comment:
Is the dealType property being changed to product?

Reply:
No in the DB it's still dealType. The "all deals" graphQL query (that gets mapped with MongoDB query in central API), returns dealType as product field. 

I had to update it here because before, it was relying on old "deals" query rather than "all deals" query.

If we want to change this to be dealType instead of product (I think either is good), should be seperate PR

Reply:
Agree for consistency should be `dealType` can be future.